lib/tensorflow/graph.rb

```diff
@@ -96,11 +96,11 @@ def make_attr_value(attribute_type, value)
   end
 
   TYPE2ENUM = {
-    DT_FLOAT: Tensorflow::TF_FLOAT,
-    DT_DOUBLE: Tensorflow::TF_DOUBLE,
-    DT_INT64: Tensorflow::TF_INT64,
-    DT_STRING: Tensorflow::TF_STRING,
-    DT_COMPLEX128: Tensorflow::TF_COMPLEX128
+    DT_FLOAT: Tensorflow::Internal::TF_FLOAT,
+    DT_DOUBLE: Tensorflow::Internal::TF_DOUBLE,
+    DT_INT64: Tensorflow::Internal::TF_INT64,
+    DT_STRING: Tensorflow::Internal::TF_STRING,
+    DT_COMPLEX128: Tensorflow::Internal::TF_COMPLEX128
```

Comment:
Is is okay, if I were to add `include Tensorflow` in` tensorflow.rb` and then remove `Tensorflow::` everywhere to do something like ?
```
    when Internal::TF_DOUBLE
      c_array = Internal::Double.new(size)
      Internal::double_reader(value, c_array, size)
    when Internal::TF_INT64
      c_array = Internal::Long_long.new(size)
      Internal::long_long_reader(value, c_array, size)
```